docs/testing-components/unit-testing/index.md

```diff
@@ -115,41 +115,29 @@ and outputs we can test basically any JavaScript function that returns a value.
 
 ## Testing React
 
-To test react we use [Enzyme](https://enzymejs.github.io/enzyme/) plus other tools you can find out about [here](../test-driven-development/index.md).
+To test react we use [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) plus other tools you can find out about [here](../test-driven-development/index.md).
 
 ```js
-const Text = (props) => {
-	return <p>{props.content}</p>;
+const Text = ({content, ...rest}) => {
+	return <p {...rest}>{content}</p>;
 }
 
 test('Should contain text', () => { 
-	const subject = shallow( 
-		<Text content='sample' /> 
+	render( 
+		<Text data-testid="test-text" content='sample' /> 
```

Comment:
Please, DO NOT USE `data-testid` when you could use other queries. This is a guidance page so you should not use it at least here.
Please read more at https://testing-library.com/docs/queries/about/#priority and fix this test.